### PR TITLE
rebase: use Rook v1.3.9 for mini-e2e jobs

### PR DESCRIPTION
**Pull request by nixpanic**
Rook v1.3.9 was released yesterday, so we can consume it for our CI jobs
now.

### CODE REVIEW

File: single-node-k8s.sh [hunk 1]
```diff
@@ -67,7 +67,7 @@ function set_env() {
     export MEMORY="8192"
     export CEPH_CSI_RUN_ALL_TESTS=true
     # downloading rook images is sometimes slow, extend timeout to 15 minutes
-    export ROOK_VERSION='v1.3.8'
+    export ROOK_VERSION='v1.3.9'
```
Comment (Madhu-1) [inline single-node-k8s.sh:70]: use v1.4.0 which is released yesterday?

Comment (nixpanic) [reply single-node-k8s.sh:70]: Not sure if that works flawlessly or if we need modifications somewhere. This is a minor update, so it should not give any regressions.

Comment (Madhu-1) [reply single-node-k8s.sh:70]: In 1.4.0 I don't see any other modifications will send a PR for that

**Comment by nixpanic** [comment]
@commitlint help

**Comment by Madhu-1** [comment]
@MergifyIO rebase

**Comment by Madhu-1** [comment]
@mergifyio rebase

**Pull request merged**